logger/module.go

```diff
@@ -92,15 +97,30 @@ func (m *loggerModule) SetBus(bus modules.Bus) {
 
 func (m *loggerModule) GetBus() modules.Bus {
 	if m.bus == nil {
-		m.logger.Fatal().Msg("Bus is not initialized")
+		m.Logger.Fatal().Msg("Bus is not initialized")
 	}
 	return m.bus
 }
 
 func (m *loggerModule) InitLogger() {
-	m.logger = m.CreateLoggerForModule(m.GetModuleName())
+	m.Logger = m.CreateLoggerForModule(m.GetModuleName())
 }
 
 func (m *loggerModule) GetLogger() modules.Logger {
-	return m.logger
+	return m.Logger
+}
+
+// SetFields sets the fields for the global logger
+func (m *loggerModule) SetFields(fields map[string]interface{}) {
+	m.Logger = m.Logger.With().Fields(fields).Logger()
```

Comment:
Feels like `Logger` should be downscoped

Reply:
Not sure if I understand, can you extend on this ?

Reply:
Why is the `Logger` parameter uppercased? Since it's something we use internally and access via a getter, shouldn't it be `.logger`?